### ISSUE TITLE
unable to publish app

### ISSUE THREAD

**Issue by johannel7** [hougaard/Simple-Object-Designer#345]
The Table Metadata Hgd does not exist. Identification fields and values: Table No.='50560'

**Comment by hougaard** [comment]
Go to the SOD Setup, use refresh symbols option (answer yes to clearing the cache).

**Comment by johannel7** [comment]
Still the same error message after clearing the cache

**Comment by hougaard** [comment]
Please post the "copy details" from the error.

**Comment by johannel7** [comment]
If requesting support, please provide the following details to help troubleshooting:

The Table Metadata Hgd does not exist. Identification fields and values: Table No.='50560'

Internal session ID: 
a81bbe82-893f-4951-84b8-2961cb0d7a74

Application Insights session ID: 
8c99e393-5348-4bd9-84fb-23f269825636

Client activity id: 
062fd417-c41b-4912-8501-22c3ff7dd45a

Time stamp on error: 
2024-02-12T21:24:27.0157394Z

User telemetry id: 
5300562a-28c8-4df5-9207-05c03310f126

AL call stack: 
"Build Extension Hgd"(CodeUnit 70310250).ExportTableExtensions line 216 - Simple Object Designer by Hougaard
"Build Extension Hgd"(CodeUnit 70310250).CreateExtensionEx line 231 - Simple Object Designer by Hougaard
"Object Designer Menu Hgd"(Page 70310263)."ExportExtension - OnAction"(Trigger) line 6 - Simple Object Designer by Hougaard

**Comment by hougaard** [comment]
Sounds like you have a Lookup Table defined as 50560 and that table no longer exists? (On a new field created).

**Comment by johannel7** [comment]
Yes I deleted things because, I have multiple companies in my environment and I did not want this anymore in this company

**Comment by johannel7** [comment]
not sure how to handle extensions for multiple companies

**Comment by hougaard** [comment]
Extensions are for the entire environment, all companies have the same code.

**Comment by johannel7** [comment]
Hi,

I created a new lookup field, how can I display the name of the field once I display the ID
For example I create a new field for the Item number but I automatically want the item name to show as a read only display
Thanks
Johanne

**Comment by hougaard** [comment]
Two options:

1. Add a (real) item name field to the table and specify that as a Lookup Field Transfer.
2. Add a Lookup FlowField to the table and use your item number field as the lookup key.

**Comment by johannel7** [comment]
Ok, I think you set me on the right path, but I would need to change the field size and I am not able to, even if my setup is set to force

***@***.***
***@***.***

**Comment by johannel7** [comment]
Here is another issue
I am trying to add a department to the opportunity card, ideally when I chose the department 01-100, I would like the flow field to say 01 Civil Engineering….
No matter how I try this, I get errors, they field is not long enough (20) to display the name, I can’t modify the number and also once I change it it goes back to zero and does not let me put it back to 20 I had to delete and start over. Now BC just closes the page and gives me an error message
***@***.***

***@***.***

***@***.***

***@***.***
***@***.***
***@***.***
***@***.***
